frontend/src/metabase-lib/v1/expressions/resolver.js

```diff
@@ -40,32 +40,32 @@ function findMBQL(op) {
   return clause;
 }
 
-// a is the type of the argument expected,
-// as defined in MBQL_CLAUSES,
-// and b is the inferred type of the argument
-const isCompatible = (a, b) => {
-  if (a === "any") {
+const isCompatible = (expectedType, inferredType) => {
```

Comment:
I renamed `a` & `b` to something more meaningful that was previously conveyed in a comment